batch/batch/front_end/validate.py

```diff
@@ -50,7 +51,7 @@
 
 SECRET_KEYS = {'namespace', 'name', 'mount_path'}
 
-RESOURCES_KEYS = {'memory', 'cpu', 'storage'}
+RESOURCES_KEYS = {'cpu', 'memory', 'storage'}
```

Comment:
is memory still a validate key in the front end?

Reply:
It has to be. Otherwise, our users will have outdated clients that submit with memory which won't work without the new release.